packages/loader/driver-utils/src/parallelRequests.ts

```diff
@@ -477,7 +490,15 @@ async function getSingleOpBatch(
 			}
 		}
 
-		await waitForConnectedState(delay);
```

Comment:
One thing that's a little odd with the current implementation is that even if we successfully receive part of the deltas, we wait with a scaling delay (same as if we had gotten an error).  This seems maybe unintentional -- probably if we successfully get part of the deltas we should just immediately turn around and request the rest.

Reply:
Took me a minute to realize this is not the case that part of the deltas was retrieved, but rather an empty set.  So this logic should be fine.  Latest does some cleanup to try to make this more obvious.

Reply:
I do not remember much :) But your description rings the bell - we run into situations where clients knows that it needs ops, but they are in transition from ordering service to storage, and thus temporarily unavailable. So client will go in the loop asking storage for ops until it gets them. I think we want to keep circling without delay.

Reply:
I thought about removing the delay, but assuming the transition from ordering service to storage takes some time, this delay helps us not blast too many requests at the storage before that happens.  Would need some research into what's a reasonable expectation for the transfer time before removing the delay.